app.vup.Chat.yaml

```diff
@@ -0,0 +1,61 @@
+app-id: app.vup.Chat
```

Comment:
Should this be `app.vup.VupChat` since the name of the app is "Vup Chat"

Reply:
Isn't that redundant? Likem it is the chat app in the vup ecosystem. We own the `vup.app` domain, and (not published yet as it is only on the dev site right now) the domain for using it in the web will be `chat.vup.app`. It seems kinda weird to have that be `vupchat.vup.app`.

Reply:
> Isn't that redundant? 

No.

The last component is NOT part of a hostname anyway. It's closely related to the name of the application.